packages/settings-library/src/settings_library/utils_r_clone.py

```diff
@@ -20,21 +20,23 @@
 }
 
 
-def _format_config(entries: dict[str, str]) -> str:
+def _format_config(entries: dict[str, str], s3_config_key: str) -> str:
     config = configparser.ConfigParser()
-    config["dst"] = entries
+    config[s3_config_key] = entries
     with StringIO() as string_io:
         config.write(string_io)
         string_io.seek(0)
         return string_io.read()
 
 
-def get_r_clone_config(r_clone_settings: RCloneSettings) -> str:
+def get_r_clone_config(r_clone_settings: RCloneSettings, *, s3_config_key: str) -> str:
     provider = r_clone_settings.R_CLONE_PROVIDER
```

Comment:
Q: would it make sense to capture the s3_config_key as part of the RCloneSettings? I see it is part of the rclone config. Moreover, can this get_rclose_config be a method of RCloneSettings?

Reply:
added some docs here